### PR TITLE
Update compose v2 reference docs, and fix relative_url filter to ignore non-.md links

### PR DESCRIPTION
**Pull request by thaJeztah**
relates to https://github.com/docker/docker.github.io/pull/13514
brings in the changes from https://github.com/docker/compose/pull/8612

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya @ndeloof ptal

File: _data/compose-cli/docker_compose_down.yaml [hunk 1]
```diff
@@ -1,63 +1,62 @@
 command: docker compose down
 short: Stop and remove containers, networks
 long: |-
-    Stops containers and removes containers, networks, volumes, and images created by ``up`.
+  Stops containers and removes containers, networks, volumes, and images created by ``up`.
```
Comment (usha-mandya) [inline _data/compose-cli/docker_compose_down.yaml:4]:
```suggestion
  Stops containers and removes containers, networks, volumes, and images created by `up`.
```

**Review by usha-mandya — APPROVED** [review]
LGTM. Noted a nit. Not a showstopper.

**Review by ndeloof — APPROVED** [review]

File: _data/compose-cli/docker_compose_down.yaml (hunk 1 above)
Comment (thaJeztah) [reply _data/compose-cli/docker_compose_down.yaml]: good catch; opened https://github.com/docker/compose/pull/8616, and updated this PR

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**